networkx/drawing/nx_pylab.py

```diff
@@ -5,15 +5,17 @@
 
 Draw networks with matplotlib.
 
-See Also
+Examples
 --------
+>>> G = nx.complete_graph(5)
+>>> nx.draw(G)
 
-matplotlib:     http://matplotlib.org/
-
-pygraphviz:     http://pygraphviz.github.io/
-
+See Also
+--------
+ - matplotlib: http://matplotlib.org/
+ - matplotlib's scatter: https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.scatter.html
+ - matplotlib's FancyArrowPatch: https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.patches.FancyArrowPatch.html
```

Comment:
```suggestion
 - :doc:`matplotlib <matplotlib:index>`
 - :func:`matplotlib.pyplot.scatter`
 - :obj:`matplotlib.patches.FancyArrowPatch`
```
We could use intersphinx for these links to make sure they remain up-to-date.